store/src/main/java/com/nytimes/android/external/store/base/impl/RealStoreBuilder.java

```diff
@@ -25,8 +25,9 @@
 public class RealStoreBuilder<Raw, Parsed, Key> {
     private final List<KeyParser> parsers = new ArrayList<>();
     private Persister<Raw, Key> persister;
-    private Cache<Key, Observable<Parsed>> memCache;
     private Fetcher<Raw, Key> fetcher;
+    private long expireAfter = 0;
+    private TimeUnit expireAfterTimeUnit = null;
```

Comment:
it might be a better approach to initialize this to a non-null value and set the expireAfter to a -1,  you can validate the input for memory() to guarantee non-nullness.  this would alleviate the need for null checking below.  another approach is to have the memory() method set a flag and use the passed in values for the builder (validate within memory() and just check flag below for inclusion...no need for validation..)